components/06-examples/beheer-basis.hbs

```diff
@@ -0,0 +1,169 @@
+            </tr>
+            <tr>
+              <td>Gemeente Delft</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Eindhoven</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Tilburg</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Breda</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Maastricht</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Amsterdam</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Utrecht</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12">
+        <nav aria-label="Pagina navigatie">
+          <ul class="pagination">
+              <li>
+                  <a href="#" aria-label="Vorige">
```

Comment:
Indenting klopt ook niet, maar dat zal wel komen door de copy/paste uit de dso-toolkit.nl markup

Reply:
Even nagelopen. Was inderdaad op basis van copy/paste bij Pagination markup in toolkit.